packages/dds/tree/src/shared-tree-core/branch.ts

```diff
@@ -77,10 +77,7 @@ export class SharedTreeBranch<TEditor extends ChangeFamilyEditor, TChange> exten
 			this.head.revision,
 		);
 
-		if (rebaseAnchors) {
-			this.changeFamily.rebaser.rebaseAnchors(this.anchors, change);
-		}
-		this.emit("onChange", change);
+		this.emitAndRebaseAnchors(change);
```

Comment:
Before, we were passing the anchorset into the editor, and the editor rebases them. So, if `applyChange` was called from the editor then we didn't want to rebase the anchors, but if `applyChange` was called by someone else then we did. Now, it doesn't matter because `applyChange` is private, but even if it were public we would still unconditionally rebase the anchors because we don't pass them into the editor anymore either (we pass a dummy AnchorSet).